### DISCUSSION TITLE
Factory.done but no web UI

### DISCUSSION THREAD

**Discussion by JVimes** [roleoroleo/yi-hack-Allwinner#432] [Q&A] [answered]
I installed y20ga_0.3.9 on my Yi 1080p Home 9FUS (firmware 12.2.0). I have Factory.done but no web UI at the IP (shown in app and router). I tried ports 80 and 8080. The log folder is empty.

Any troubleshooting tips?

**Comment by roleoroleo** [comment]
Are you able to login with ssh?
Or maybe with telnet?

**Comment by JVimes** [comment, marked as answer]
Oops, camera was on guest network, computer on main network 😉
